examples/with-markdoc/next.config.js

```diff
@@ -0,0 +1,12 @@
+const withMarkdoc = require('@markdoc/next.js')
+// const path = require('path')
+
+module.exports = withMarkdoc({
+  /**
+   * @see https://markdoc.io/docs/nextjs#options
+   */
+  mode: 'server', // 'static' | 'server',
+  //schemaPath: path.join(__dirname, 'markdoc')
```

Comment:
Please remove the options here. We want `static` (the default).  We also should remove the commented out `schemaPath`

I like the `@see` comment you have here